egs2/TEMPLATE/asr1/pyscripts/utils/evaluate_whisper_inference.py

```diff
@@ -88,7 +90,7 @@ def inference(
 
     # 3. Build data-iterator
     info_list = []
-    wavscp = open(data_path_and_name_and_type, "r", encoding="utf-8")
+    wavscp = open(key_file, "r", encoding="utf-8")
```

Comment:
`key_file` can contain only a subset of utterances due to the use of multiple jobs. `data_path_and_name_and_type` can contain all the data.